examples/README.md

```diff
@@ -39,7 +39,7 @@ or `local-gadget` directly.
     packages to filter events by container.
     - [trace/exec](gadgets/withfilter/trace/exec/): traces creation of
       new processes inside a particular container.
-  - [parser](gadgets/parser/): Examples showing how to use a parser to
-    format the events.
-    - [trace/exec](gadgets/parser/trace/exec/): traces creation of
+  - [formatter](gadgets/formatter/): Examples showing how to use a formatter to
+    print events in a column format.
+    - [trace/exec](gadgets/formatter/trace/exec/): traces creation of
```

Comment:
> Parser is a concept we won't be using anymore.
>
I am not completely sure, see https://github.com/kinvolk/inspektor-gadget/pull/980#discussion_r982391991. In any case, we think the best way to go is to use directly the formatter for the examples. WDYT?

Reply:
I changed the commit message. I agree it's better to use the formatter on the examples.